src/kolibri_tools/initialization.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+import sys
+
+import pew.ui
+
+
+def setup_env():
+    script_dir = os.path.dirname(os.path.abspath(__file__))
+    root_dir = os.path.abspath(os.path.join(script_dir, '..'))
+    logging.info("service script root_dir = {}".format(root_dir))
+
+    sys.path.append(root_dir)
+    sys.path.append(os.path.join(root_dir, "kolibri", "dist"))
+
+    os.environ["DJANGO_SETTINGS_MODULE"] = "kolibri_tools.django_app_settings"
+
+    # TODO: before shipping the app, make this contingent on debug vs production mode
+    os.environ["KOLIBRI_RUN_MODE"] = "pew-dev"
```

Comment:
Do we want to remove this entirely, or set it to something else? I think Android is the only platform where the user cannot set `KOLIBRI_RUN_MODE` themselves easily. 

Another possibility would be to use a versioning scheme where official releases can be clearly distinguished. (e.g. using alpha or beta tags in non-production builds), and set KOLIBRI_RUN_MODE in the app when those tags are found in the version string. This would allow us to strip out pre-release builds from stats regardless of the machine they were run on.

Reply:
We should remove it entirely, yeah. We do already detect and automatically mark a "dev" runmode on the pingbacks for non-production releases, so we can just lean on that.